README.md

```diff
@@ -95,9 +81,9 @@ android {
 
 ### Using AWS Mobile Client
 
-Authentication in Amplify for Android is dependent on AWS Mobile Client. Please refer to the [this documentation](https://aws-amplify.github.io/docs/android/authentication) for details.
+Amplify relies on the AWS Mobile Client for authentication with AWS services. Please refer to the [this documentation](https://aws-amplify.github.io/docs/android/authentication) for details.
```

Comment:
https://docs.amplify.aws/lib/auth/getting-started?platform=android

Auto redirects already, but might as well update it

Reply:
updated